connection/database.feature

```diff
@@ -126,13 +126,17 @@ Feature: Connection Database
     Then connection does not have any database
 
 
+  # TODO: currently throws in @After; re-enable when we are able to check if sessions are alive (see client-java#225)
```

Comment:
Deleting the database causes the sessions to disappear remotely, but the client has no way of knowing this.

It boldly attempts to close all the sessions in the scenario tear-down `@After`, and this throws errors because they don't exist, causing the scenario to unavoidably fail.

Our proposed solution, listed in the linked issue, is to add a mechanism for checking if a session is still alive. We will be able to piggyback on our Pulse mechanism, which happens to do exactly this.